typescript/cli/src/submit/submitCommand.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  PopulatedTransaction,
+  PopulatedTransactionSchema,
+} from '@hyperlane-xyz/sdk';
+import { assert } from '@hyperlane-xyz/utils';
+
+import { WriteCommandContext } from '../context/types.js';
+import { logGray } from '../logger.js';
+import { readYamlOrJson } from '../utils/files.js';
+
+import { getSubmitterBuilder } from './submit.js';
+
+export async function runSubmit({
+  context,
+  transactionsFilepath,
+}: {
+  context: WriteCommandContext;
+  transactionsFilepath: string;
+}) {
+  const { submissionStrategy, chainMetadata, multiProvider, isDryRun } =
+    context;
+
+  assert(
+    submissionStrategy,
+    'Submission strategy required to submit transactions.\nPlease create a submission strategy, e.g. ./strategy.yaml.',
+  );
+
+  const chain = submissionStrategy.chain;
+  const protocol = chainMetadata[chain].protocol;
+  const submitterBuilder = await getSubmitterBuilder<typeof protocol>({
+    submitterMetadata: submissionStrategy.submitter,
+    transformersMetadata: submissionStrategy.transforms ?? [],
+    multiProvider,
+    chain: submissionStrategy.chain,
+    isDryRun,
+  });
+  const transactions = getTransactions(transactionsFilepath);
+  const transactionReceipts = await submitterBuilder.submit(...transactions);
```

Comment:
can we prompt the user to confirm the transactions and strategy before submitting?

Reply:
unless we intend to capture more information from the user, i think logging is sufficient (instead of waiting for a `Y` or `enter` from user just to continue)